src/creature.cpp

```diff
@@ -685,7 +685,8 @@ void Creature::deal_projectile_attack( Creature *source, dealt_projectile_attack
         if( z ) {
             if( !proj.get_drop().is_null() ) {
                 z->add_effect( effect_tied, 1_turns, num_bp, true );
-                z->tied_item = proj.get_drop();
+                item drop_item = proj.get_drop();
```

Comment:
`drop_item` is not used at all (see #37053)

Reply:
Probably got lost in the refactor for value pointers.